src/trading/NewOrders.sol

```diff
@@ -0,0 +1,338 @@
+        return orders[_orderId].fxpAmount;
+    }
+
+    function getPrice(bytes20 _orderId, Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public constant returns (uint256) {
+        return orders[_orderId].fxpPrice;
+    }
+
+    function getOrderOwner(bytes20 _orderId, Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public constant returns (address) { 
+        return orders[_orderId].owner;
+    }
+
+    function getOrderSharesEscrowed(bytes20 _orderId, Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public constant returns (uint256) { 
+        return orders[_orderId].fxpSharesEscrowed;
+    }
+
+    function getOrderMoneyEscrowed(bytes20 _orderId, Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public constant returns (uint256) { 
+        return orders[_orderId].fxpMoneyEscrowed;
+    }
+
+    function getVolume(IMarket _market) public constant returns (uint256) { 
+        return marketOrderData[_market].volume;
+    }
+
+    function getLastOutcomePrice(IMarket _market, uint8 _outcome) public constant returns (uint256) { 
+        return marketOrderData[_market].prices[_outcome];
+    }
+
+    function getBetterOrderId(bytes20 _orderId, Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public constant returns (bytes20) { 
+        return orders[_orderId].betterOrderId;
+    }
+
+    function getWorseOrderId(bytes20 _orderId, Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public constant returns (bytes20) {
+        return orders[_orderId].worseOrderId;
+    }
+
+    function getGasPrice(bytes20 _orderId, Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public constant returns (uint256) { 
+        return orders[_orderId].gasPrice;
+    }
+
+    function getBestOrderId(Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public constant returns (bytes20) { 
+        return bestOrder[getBestOrderWorstOrderHash(_market, _outcome, _type)];
+    }
+
+    function getWorstOrderId(Trading.TradeTypes _type, IMarket _market, uint8 _outcome) public constant returns (bytes20) { 
+        return worstOrder[getBestOrderWorstOrderHash(_market, _outcome, _type)];
+    }
+
+    // FIXME: Currently, getOrderIdHash(), getBestOrderWorstOrderHash(), and makeOrder.makeOrder() are creating ripemd160 hash values of type bytes20.  This should probably be changed to create sha3/sha256 hash values of type bytes32.
+    function getOrderIdHash(Trading.TradeTypes _type, IMarket _market, uint256 _fxpAmount, uint256 _fxpPrice, address _sender, uint256 _blockNumber, uint256 _outcome, uint256 _fxpMoneyEscrowed, uint256 _fxpSharesEscrowed) public constant returns (bytes20) {
+        return ripemd160(_type, _market, _fxpAmount, _fxpPrice, _sender, _blockNumber, _outcome, _fxpMoneyEscrowed, _fxpSharesEscrowed);
+    }
+
+    function getBlockNumber() public constant returns (uint256) {
```

Comment:
This isn't used in this file, nor does it appear to be used anywhere else.  We should delete it.

Reply:
It's used in test_orders.py for calling orders.getOrderIdHash().  I figured having the function getOrderIdHash might also be useful in some scenarios if we want to get the orderId hash without actually calling saveOrder().  If you still think I should remove it, how should I be setting the better/worse order IDs that are used for the asserts in test_orders.py?